consoles/ssh_screen.pm

```diff
@@ -83,6 +83,8 @@ sub type_string ($self, $nargs) {
             select(undef, undef, undef, 0.1);
         } elsif ($chunk < 0) {
             # Old Net::SSH2 error signaling
+            print "$chunk\n";
+            print LIBSSH2_ERROR_EAGAIN"\n";
```

Comment:
This is a debugging leftover, right?

Reply:
Yes, I need to do a refactor of the code and the commits because they are really bad